### ISSUE TITLE
subscribe to notifications

### ISSUE THREAD

**Issue by piamancini** [opencollective/backyourstack#18]
feedback: 

> One feature that could make this reusable is to be able to subscribe or get notified when “projects requiring funding” changes

**Comment by cuiki** [comment]
Hi @piamancini , @xdamman , @znarf , here are some options for this:

1. Monthly report field placed on sidebar: https://tafka.d.pr/ihqKKK
2. Monthly report field placed on sidebar with a bit more prominence: https://tafka.d.pr/SHBXAA
3. Monthly report field below the open source projects: https://tafka.d.pr/MGGGis

**Comment by xdamman** [comment]
I like option 1. Can you show on mobile and the state after the user subscribes?

**Comment by piamancini** [comment]
I like option 1 too. 
I'd make the copy simpler: 
Stay up to date: 
Receive a notification when one of your dependencies is ready to receive funding and a monthly report on your stack's progress, goals and updates.

**Comment by cuiki** [comment]
Hi, the text changes have been applied on InVision prototype. I'll show the state right after subscribing in a few minutes, though I'd like to clarify: After the user enters the email address we'll show a confirmation either modal or bubble or something right there, but it should be temporary to bring back the field. A user might want to add a different email to subscribe to the same report. Does that make sense?

**Comment by xdamman** [comment]
No modal, maybe we just show a confirmation text below the input field and
reset that field.
Let's keep things simple and lightweight.

**Comment by cuiki** [comment]
Right, yeah just to confirm that the filed will be reset after the confirmation.
